app/dashboard/views.py

```diff
@@ -1315,6 +1315,40 @@ def profile_filter_activities(activities, activity_name):
     return activities.filter(activity_type=activity_name)
 
 
+def handle_kudos_pinning_helper(profile, pk, position):
+    """A helper function pin kudos to a profile.
+    Presumes that user is authenticated as this profile before it takes action
+
+    Args:
+        pk - a KudosTransfer id.
+        position - int - a position 1 -5.
+
+    Returns:
+        nothing
+
+    """
+    if not pk.isdigit():
+        return
+    if not position.isdigit():
+        return
+    position = int(position)
+    if position < 1 or position > 5:
```

Comment:
What do you think of using a `settings.py` entry for the max number of pins? Its mentioned a few times throughout the code and would be good to avoid having a magic number